old_tests/test_lib.py

```diff
@@ -270,3 +270,7 @@ def test_does_not_coalesce_info_lines(self):
         err = ('ERROR: You have made a terrible mistake\n'
                '  2015-05-12 14:12:12,720 INFO: why would you ever do that\n')
         self.assertGreppedLog(err, [['ERROR: You have made a terrible mistake']])
+
+    def test_ignore_error_as_a_part_of_word(self):
```

Comment:
FYI, those test we are not running in CCM CI

but if it was enough for your needs...

Reply:
It looked like the most logical place for this test. It's a pity that it's not being run in CI, should I move it somewhere else? 
All tests from `TestErrorLogGrepping` seem to pass.

Reply:
never mind, it's a refactoring that doesn't belong to this PR...